### PR TITLE
Add configure-openshift-monitoring playbook

### PR DESCRIPTION
**Pull request by akrzos**

### CODE REVIEW

**Review by smalleni — COMMENTED** [review]
Looks good, so the assumption is something/someone would have labelled the nodes before running this?

**Comment by akrzos** [comment]
> Looks good, so the assumption is something/someone would have labelled the nodes before running this?

Yup, I am putting in a pr for a node label playbook.  Idea is to list nodes and the labels for them so we can label any node we want separate from a playbook which applies something like localstorage or kafka or prometheus configuration etc...

**Comment by smalleni** [comment]
LGTM

**Review by smalleni — APPROVED** [review]

File: ansible/group_vars/all.yml [hunk 1]
```diff
@@ -77,7 +77,15 @@ shiftstack_local_storage_node_selector: beta.kubernetes.io/instance-type=nvme-8
...
 shiftstack_thin_volume_size: 140g
-  
+
+################################################################################
+# Configure OpenShift Monitoring playbook vars
+################################################################################
+openshift_prometheus_retention_period: 30d
+openshift_prometheus_node_selector: "localstorage: true"
+openshift_prometheus_storage_class: local-sc
+openshift_prometheus_storage_size: 50Gi
```
Comment (smalleni) [inline ansible/group_vars/all.yml:87]: Just for the record, with local-storage I see the entire PV being used irrespective of what is specified here.

**Pull request merged**